content/blog/introducing-kube2pulumi/index.md

```diff
@@ -0,0 +1,417 @@
+                "update",
+                "watch",
+            ],
+        },
+    ],
+});
+const pulumi_kubernetes_operatorRoleBinding = new kubernetes.rbac.v1.RoleBinding("pulumi_kubernetes_operatorRoleBinding", {
+    kind: "RoleBinding",
+    apiVersion: "rbac.authorization.k8s.io/v1",
+    metadata: {
+        name: "pulumi-kubernetes-operator",
+    },
+    subjects: [{
+        kind: "ServiceAccount",
+        name: "pulumi-kubernetes-operator",
+    }],
+    roleRef: {
+        kind: "Role",
+        name: "pulumi-kubernetes-operator",
+        apiGroup: "rbac.authorization.k8s.io",
+    },
+});
+const pulumi_kubernetes_operatorServiceAccount = new kubernetes.core.v1.ServiceAccount("pulumi_kubernetes_operatorServiceAccount", {
+    apiVersion: "v1",
+    kind: "ServiceAccount",
+    metadata: {
+        name: "pulumi-kubernetes-operator",
+    },
+});
+```
+
+## Superpowers
+
+Now that we've got some Kubernetes resources defined in real code, we can start working on them like any other software
+project. First, we define a variable for the operator name and then swap out hardcoded string references to the other
+resource names. With these changes, it doesn't require any thought to keep these resources in sync! You can later decide
+to change the name in one place, and every other reference will update automatically as well.
+
+```typescript
+const operatorName = "pulumi-kubernetes-operator";
+
+const pulumi_kubernetes_operatorServiceAccount = new kubernetes.core.v1.ServiceAccount("pulumi_kubernetes_operatorServiceAccount", {
+    apiVersion: "v1",
+    kind: "ServiceAccount",
+    metadata: {
+        name: operatorName,
+    },
+});
+const pulumi_kubernetes_operatorRoleBinding = new kubernetes.rbac.v1.RoleBinding("pulumi_kubernetes_operatorRoleBinding", {
+    kind: "RoleBinding",
+    apiVersion: "rbac.authorization.k8s.io/v1",
+    metadata: {
+        name: operatorName,
+    },
+    subjects: [{
+        kind: "ServiceAccount",
+        name: pulumi_kubernetes_operatorServiceAccount.metadata.name,
+    }],
+    roleRef: {
+        kind: "Role",
+        name: pulumi_kubernetes_operatorRole.metadata.name,
+        apiGroup: "rbac.authorization.k8s.io",
+    },
+});
+```
+
+This simple refactoring is just the tip of the iceberg. At this point, you could:
+
+- wrap these resources in a class to
+abstract away the low-level details,
+- add configuration values to the stack to make it easy to tweak certain values,
+- or even add custom logic for Pulumi to run after a particular resource is created.
+
+Pulumi gives you superpowers, and
+you decide how to use them!
+
+## Learn More
+
+If you'd like to learn about Pulumi and how to manage your
+infrastructure and Kubernetes through code, [get started today]({{< relref "/docs/get-started" >}}). Pulumi is open
+source and free to use.
+
+For further examples on how to use Pulumi to create Kubernetes
+clusters, or deploy workloads to a cluster, check out the rest of the
```

Comment:
Nit - Perhaps call out that we are only scratching the surface with these "super powers" and that we welcome feedback and feature requests?